local/running_environment.ex

```diff
@@ -93,14 +95,17 @@ defmodule AntikytheraLocal.RunningEnvironment do
   defp fetch_current_version(app_name_str) do
     url = "http://#{AntikytheraCore.Cmd.hostname()}:8080/versions"
     token = File.read!(@system_info_access_token_path)
-    %Httpc.Response{status: 200, body: body} = Httpc.get!(url, %{"authorization" => token})
-    String.split(body, "\n", trim: true)
-    |> Enum.find_value(fn line ->
-      case String.split(line) do
-        [^app_name_str, v] -> v
-        _                  -> nil
-      end
-    end)
+    case Httpc.get(url, %{"authorization" => token}) do
+      {:ok, %Httpc.Response{status: 200, body: body}} ->
+        String.split(body, "\n", trim: true)
+        |> Enum.find_value(fn line ->
```

Comment:
Use `Enum.find_value/3` instead of `/2` to correctly return `{:error, :not_found}`.
Then we can combine 2 operations (getting HTTP response and extracting version string) by `R.bind/2` as follows:
```ex
Httpc.get(url, %{"authorization" => token})
|> R.bind(fn %Httpc.Response{status: 200, body: body} ->
  String.split(body, "\n", trim: true)
  |> Enum.find_value({:error, :not_found}, fn line ->
    case String.split(line) do
      [^app_name_str, v] -> {:ok, v}
      _                  -> nil
    end
  end)
end)
```
You may (of course) freely make private function(s) to decompose this function.

Reply:
Ah, you jut want this code to crash when `app_name_str` doesn't exist in the body.
Then, (although it's about the issue before this PR) let's make the intention explit:

```ex
|> Enum.find(fn line ->
  ...
end) || raise "#{app_name_str} not found in the body!"
```

Here precedence of binary operators is a bit trickey; you may define some helpers here.
(We actually want `Enum.find!`, `Enum.find_value!` etc. here.)

Reply:
fixed in b114f76

Now I have some questions.
- Should we consider the possibility that `line` does not start with `app_name_str`?
For example: `fn line -> String.trim_leading(line) |> String.starts_with?(app_name_str) end`

- Now, I have passed `app_name_str` as an argument for error output only.
However, this seems a subtle way.
Is it OK in this state? Or should we improve?

Reply:
First of all, due to my misleading comment (sorry), things got a bit too complicated.
Now we don't need `R.bind/2`; what we want is `R.map/2`.

> Should we consider the possibility that line does not start with app_name_str?

In this case absolutely not!
The response body format is defined by us [here](https://github.com/access-company/antikythera/blob/master/core/handler/system_info_exporter.ex#L37).
Any discrepancies from our expectations should make the tests fail.